test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -63,19 +63,23 @@ while [ "$1" != "" ]; do
     shift
 done
 
-# PULL_PULL_SHA is empty whne Pros/Tide tests the batches.
-# PULL_BASE_SHA cannot be used here as it still points to master tip in that case.
-PULL_PULL_SHA="${PULL_PULL_SHA:-$(git rev-parse HEAD)}"
+# This is merged commit's SHA.
+COMMIT_SHA="$(git rev-parse HEAD)"
 
-# Variables
-GCR_IMAGE_BASE_DIR=gcr.io/${PROJECT}/${PULL_PULL_SHA}
-TEST_RESULTS_GCS_DIR=gs://${TEST_RESULT_BUCKET}/${PULL_PULL_SHA}/${TEST_RESULT_FOLDER}
+# Paths are using commit sha, instead of pull sha, because tests may be rerun with the same PR
```

Comment:
`PULL_PULL_SHA` is expected to be exactly that - the SHA of a merge commit between the current master tip and the tip of the branch.